src/main/java/org/opentripplanner/routing/alternativelegs/AlternativeLegs.java

```diff
@@ -45,14 +45,25 @@ public static List<ScheduledTransitLeg> getAlternativeLegs(
     TransitService transitService,
     boolean searchBackward,
     AlternativeLegsFilter filter
+  ) {
+    return getAlternativeLegs(leg, numberLegs, transitService, searchBackward, filter, false);
+  }
+
+  public static List<ScheduledTransitLeg> getAlternativeLegs(
+    Leg leg,
+    Integer numberLegs,
+    TransitService transitService,
+    boolean searchBackward,
+    AlternativeLegsFilter filter,
+    boolean exactOriginStop
```

Comment:
This should probably be used for both origin and destination?

Reply:
Yes, the exact destination stop argument could also be useful in the future even if I don't have a need for it right now. Will add that option